mod/wikirate/set/right/discussion.rb

```diff
@@ -2,4 +2,11 @@
   view :editor do |args|
     text_area :comment, rows: 3
   end
+  view :comment_box,
+       denial: :blank, tags: :unknown_ok,
+       perms: ->(r) { r.card.ok? :comment } do |_args|
+    <<-HTML
+      <div class="comment-box nodblclick">#{comment_form}</div>
+    HTML
+  end
```

Comment:
`comment_box` is a view in a all set. No need to repeat it here. 
If it wasn't in all you can write something like `view :comment_box, mod: All::Comment::Format` to refer to the other view